sfaira/data/dataloaders/loaders/d10_1016_j_cell_2020_08_001/human_blood_2020_na_schulteschrepping_001.yaml

```diff
@@ -0,0 +1,159 @@
+dataset_structure:
+    dataset_index: 1
+    sample_fns:
+        - "Schulte-Schrepping_2020_COVID19_10x_Neutrophils_processed_cellXgene.h5ad"
+        - "Schulte-Schrepping_2020_COVID19_10x_PBMC_processed_cellXgene.h5ad"
+        - "Schulte-Schrepping_2020_COVID19_Rhapsody_freshPBMC-WB_processed_cellXgene.h5ad"
+        - "Schulte-Schrepping_2020_COVID19_Rhapsody_Monocytes_processed_cellXgene."
+        - "Schulte-Schrepping_2020_COVID19_Rhapsody_Neutrophils_processed_cellXgene.h5ad"
+        - "Schulte-Schrepping_2020_COVID19_Rhapsody_PBMC_processed_cellXgene.h5ad"
+dataset_wise:
+    author: "Schulte-Schrepping, Jonas"
+    default_embedding:
+        Schulte-Schrepping_2020_COVID19_10x_Neutrophils_processed_cellXgene.h5ad: "X_umap" 
+        Schulte-Schrepping_2020_COVID19_10x_PBMC_processed_cellXgene.h5ad: "X_umap"
+        Schulte-Schrepping_2020_COVID19_Rhapsody_freshPBMC-WB_processed_cellXgene.h5ad: "X_umap"
+        Schulte-Schrepping_2020_COVID19_Rhapsody_Monocytes_processed_cellXgene.h5ad: "X_umap"
+        Schulte-Schrepping_2020_COVID19_Rhapsody_Neutrophils_processed_cellXgene.h5ad: "X_umap"
+        Schulte-Schrepping_2020_COVID19_Rhapsody_PBMC_processed_cellXgene.h5ad: "X_umap"
```

Comment:
If it s the same value in each of the data sets, you can also just state that value instead of listing all of them: `default_embedding: "X_umap"` etc for other attributes!